python/paddle/fluid/layers/nn.py

```diff
@@ -352,6 +353,38 @@ def fc(input,
     return helper.append_activation(pre_activation)
 
 
+def center_loss(input,
+                labels,
+                num_classes,
+                alpha,
+                param_attr,
+                update_center=True):
+    helper = LayerHelper('center_loss', **locals())
+    dtype = helper.input_dtype()
+    centers_shape = [num_classes, input.shape[1]]
+    centers_param = helper.create_parameter(
+        attr=helper.param_attr, shape=centers_shape, dtype=dtype)
```

Comment:
the input param_attr is not the helper.param_attr, i think it is what elephant mentioned

Reply:
yes. used to initialize the centers